### DISCUSSION TITLE
ct.templates

### DISCUSSION THREAD

**Discussion by giscus** [ct-js/docs.ctjs.rocks#109] [Pages' discussions]
# ct.templates

This object allows you to create new Copies and manipulate them. ct.templates.copy(template, x, y, exts) Creates a new Copy of a given template. template must be a string — the ...

https://docs.ctjs.rocks/ct.templates.html

**Comment by Lozamded** [comment]
theres is a way to get all templates of a collision group?

**Reply by CosmoMyzrailGorynych** [reply]
No, there is not.
